graylog2-server/src/main/java/org/graylog2/shared/rest/resources/system/RemoteDataNodeStatusResource.java

```diff
@@ -33,4 +36,9 @@ public interface RemoteDataNodeStatusResource {
     @GET("/logs/stderr")
     Call<List<String>> opensearchStdErr();
 
+    @Streaming
+    @Headers({"Accept: */*"})
```

Comment:
why not use `text/plain` as this is what is expected?

Reply:
copy pasted from graylog-server logs, fixed now, thanks!